browser/src/layer/tile/CalcTileLayer.js

```diff
@@ -932,6 +933,39 @@ window.L.CalcTileLayer = window.L.CanvasTileLayer.extend({
 		else if (e.commandName === 'PivotTableFilterInfo') {
 			app.calc.pivotTableFilterCell = { 'row': e.state.row, 'column': e.state.column };
 		}
+		else if (e.commandName === 'TableAutoFillInfo') {
+			this._onTableAutoFillStateChanged(e.state.rectangle);
+		}
+	},
+
+	_onTableAutoFillStateChanged: function (textMsg) {
+		var tablefillMarkerSection = app.sectionContainer.getSectionWithName(app.CSections.TableFillMarker.name);
+		if (textMsg.match('EMPTY')) {
+			if (tablefillMarkerSection)
+				tablefillMarkerSection.calculatePositionViaCorePixels(null);
+			this._tableAutoFillAreaPixels = null;
```

Comment:
I guess `_tableAutoFillAreaPixels` can also be a local variable of `TableFillMarkerSection` class. But it's not a big thing.